integrationTests/testInitializer.go

```diff
@@ -486,7 +486,7 @@ func ProposeBlock(nodes []*TestProcessorNode, idxProposers []int, round uint64)
 			continue
 		}
 
-		body, header, _ := n.ProposeBlock(round)
+		body, header, _ := n.ProposeBlock(round, round)
```

Comment:
can you use a separate variable for nonce?

Reply:
Done